oak-core/src/main/java/org/apache/jackrabbit/oak/Oak.java

```diff
@@ -801,7 +802,8 @@ private ContentRepository createNewContentRepository() {
                 queryEngineSettings.unwrap(),
                 indexProvider,
                 securityProvider,
-                new AggregatingDescriptors(t)) {
+                new AggregatingDescriptors(t),
+                newFeature("OAK-10147", whiteboard)) {
```

Comment:
elsewhere we used a slightly different naming convention (eg "FT_PREFETCH_OAK-9780"). Is this here the final name?